stix2/test/test_data_sources.py

```diff
@@ -212,39 +213,39 @@ def test_add_get_remove_filter(ds):
         Filter('created', '=', object()),
     ]
 
-    assert len(ds.filters) == 0
+    assert len(ds._filters) == 0
```

Comment:
The fact that this information is needed outside the class means we should probably not make it private.

Reply:
Even if just for test purposes?

Reply:
It may be useful for users to be able to check the current list of filters.